When building without berkeley db support, don't die when db.h is not found.

--- a/makedefs
+++ b/makedefs
@@ -273,18 +273,17 @@ case "$SYSTEM.$RELEASE" in
 		# Postfix no longer needs DB 1.85 compatibility
 		if [ -f /usr/include/db.h ]
 		then
-		    : we are all set
+		    SYSLIBS="-ldb"
 		elif [ -f /usr/include/db/db.h ]
 		then
 		    CCARGS="$CCARGS -I/usr/include/db"
+		    SYSLIBS="-ldb"
 		else
 		    # No, we're not going to try db1 db2 db3 etc.
 		    # On a properly installed system, Postfix builds
 		    # by including <db.h> and by linking with -ldb
 		    echo "No <db.h> include file found." 1>&2
-		    echo "Install the appropriate db*-devel package first." 1>&2
-		    echo "See the RELEASE_NOTES file for more information." 1>&2
-		    exit 1
+		    echo "Building without Berkeley DB support." 1>&2
 		fi
 		# GDBM locks the DBM .pag file after open. This breaks postmap.
 		# if [ -f /usr/include/gdbm-ndbm.h ]
@@ -296,7 +295,6 @@ case "$SYSTEM.$RELEASE" in
 		#     CCARGS="$CCARGS -DHAS_DBM -DPATH_NDBM_H='<gdbm/ndbm.h>'"
 		#     GDBM_LIBS=gdbm
 		# fi
-		SYSLIBS="-ldb"
 		for name in nsl resolv $GDBM_LIBS
 		do
 		    for lib in /usr/lib64 /lib64 /usr/lib /lib
@@ -352,19 +350,17 @@ EOF
     Linux.3*)	SYSTYPE=LINUX3
 		if [ -f /usr/include/db.h ]
 		then
-		    : we are all set
+		    SYSLIBS="-ldb"
 		elif [ -f /usr/include/db/db.h ]
 		then
 		    CCARGS="$CCARGS -I/usr/include/db"
+		    SYSLIBS="-ldb"
 		else
 		    # On a properly installed system, Postfix builds
 		    # by including <db.h> and by linking with -ldb
 		    echo "No <db.h> include file found." 1>&2
-		    echo "Install the appropriate db*-devel package first." 1>&2
-		    echo "See the RELEASE_NOTES file for more information." 1>&2
-		    exit 1
+		    echo "Building without Berkeley DB support." 1>&2
 		fi
-		SYSLIBS="-ldb"
 		for name in nsl resolv
 		do
 		    for lib in /usr/lib64 /lib64 /usr/lib /lib
@@ -381,20 +377,18 @@ EOF
 		# Postfix no longer needs DB 1.85 compatibility
 		if [ -f /usr/include/db.h ]
 		then
-		    : we are all set
+		    SYSLIBS="-ldb"
 		elif [ -f /usr/include/db/db.h ]
 		then
 		    CCARGS="$CCARGS -I/usr/include/db"
+		    SYSLIBS="-ldb"
 		else
 		    # No, we're not going to try db1 db2 db3 etc.
 		    # On a properly installed system, Postfix builds
 		    # by including <db.h> and by linking with -ldb
 		    echo "No <db.h> include file found." 1>&2
-		    echo "Install the appropriate db*-devel package first." 1>&2
-		    echo "See the RELEASE_NOTES file for more information." 1>&2
-		    exit 1
+		    echo "Building without Berkeley DB support." 1>&2
 		fi
-		SYSLIBS="-ldb"
 		for name in nsl resolv
 		do
 		    for lib in /usr/lib64 /lib64 /usr/lib /lib
